src/hooks/useLedger.ts

```diff
@@ -78,25 +75,16 @@ const useLedger = () => {
         display,
         accountIndex,
       )
-      let balance: number | undefined
-      try {
-        const { data: accountData } = await getAccount({
-          variables: { address },
-        })
-        balance = accountData?.account?.balance
-      } catch (e) {
-        const error = e as ApolloError
-        throw new Error(`Api failure - ${error.message}`)
-      }
-
+      const balance = 0
+      console.error('LEDGER NOT CURRENTLY SUPPORTED')
```

Comment:
I wonder if we should take the ledger option out on import Account?

Reply:
We have a separate ticket for that